esp/services/ws_workunits/ws_workunitsHelpers.cpp

```diff
@@ -315,6 +315,15 @@ void WsWuInfo::setLogTimeRange(LogAccessConditions& logFetchOptions, unsigned wu
 }
 
 #ifdef _CONTAINERIZED
+void WsWuInfo::sendImportedWorkunitComponentLog(const char* logFile, CHttpResponse* response)
```

Comment:
is there a reason this can't compile in non-containerized too?

In general we should compile in containerized code, so that it gets (compile) tested at least,
but also because things like remote logging, is something we that could equally apply to bare-metal too in the future (given a suitable configuration etc.)

Reply:
It is inside the _CONTAINERIZED because it is used for ComponentLog and, for now, the ComponentLog is only for containerized environment. I will move it out.

Reply:
not new: but there is other code which is to do with remote logging, which really shouldn't be in a _CONTAINIZED only (for same reasons as above).
Unless there's some reason that it only compiles in containized, this may be a good time to remove other #ifdef _CONTAINERIZED ?